actions/templates/install/index.js

```diff
@@ -171,10 +181,16 @@ async function main (params) {
         if (apiFlowType.toLowerCase() !== CREDENTIAL_FLOW_TYPE_ADOBEID || apiCredentialType.toLowerCase() !== credentialType.toLowerCase()) {
           continue;
         }
+        // extract api license config from the api info map
+        let apiLicenseConfigs = [];
+        const apiInfo = mapApiCodeToApiInfo[api.code];
+        if (apiInfo && apiInfo.credentialType.toLowerCase() === apiCredentialType.toLowerCase() && apiInfo.flowType.toLowerCase() === apiFlowType.toLowerCase()) {
```

Comment:
?? is a thing ... so is apiInfo?.credentialType?.toLowerCase()

it seems like this could be simplified a lot, can we ensure it is lower case before making all these conditional comparisons?